### PR TITLE
Creating Brazilian Portuguese version of resource files.

### PR DESCRIPTION
**Pull request by paulluscastro**
Despite Brazilian Portuguese and European Portuguese being mutually inteligible the way native speakers express themselves is different.

The current Portuguesse translation sounds awkard to Brazilian natives, this is why I decided to contribute with the project.

### CODE REVIEW

**Comment by ravibpatel** [comment]
There are no resource files for DownloadUpdateDialog and RemindLaterForm. Does translation for them is decent and it doesn't require it? 
Let me know so I can merge the request.

**Comment by paulluscastro** [comment]
I didn't notice those files. I will do their translation and commit again. This is my first pull request ever, how shall I proceed?

**Comment by ravibpatel** [comment]
I think you can commit new files and it will show up in the pull request.

File: AutoUpdater.NET/Properties/Resources.pt-BR.resx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
+  <data name="clock_go" type="System.Resources.ResXFileRef, System.Windows.Forms">
```
Comment (ravibpatel) [inline AutoUpdater.NET/Properties/Resources.pt-BR.resx]: Please remove duplicate resource entries like this.

**Pull request merged**